lib/repl.js

```diff
@@ -1111,7 +1111,30 @@ function complete(line, callback) {
     }
 
     completionGroupsLoaded();
+  } else if (match = line.match(/fs.*\( *["'](.*)/)) {
+    try {
```

Comment:
Please move this `try` inside of the inner `catch`. That way it's less indentation. It is only required in case the second `fs` call in the catch fails.